modules/ROOT/pages/afm-add-schedule.adoc

```diff
@@ -5,43 +5,57 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can add a schedule to an existing monitor if either of the following two conditions is true:
+You can add a schedule to an existing monitor if either of the following two conditions are true:
 
-* You plan to run the test associated with the schedule in a public location and the allocation of schedules for your organization in Anypoint Platform is not exhausted.
+* You plan to run the test associated with the schedule in a public location, and the allocation of schedules for your Anypoint Platform organization has not been exhausted.
 * You plan to run the test associated with the schedule in a private location.
 
-When you add a schedule to a monitor, you can use the schedule to run any of the versions of the test for that monitor.
-
 == Procedure
 
-There are two different methods that you can use to add a schedule.
-
-To add a schedule from the *Schedules* section of a monitor:
-
 . Select the monitor in which you want to add a schedule.
-. Click the *Schedules* tab.
-. Click *Add Schedule*.
-. In the Add Schedule dialog, specify this information:
+. Click *Add Schedule* from the *Schedules* table.
+. In the Add Schedule dialog, specify the following:
 +
-* The test version that you want the schedule to run
-* The location to run the test from
-* The frequency at which to run the test
-. Click *Add*.
+* The location in which to run the test
+* The interval at which to run the test:
++
+|===
+|Interval |Required Information |Defintion
 
-To add a schedule from the list of schedules that use a particular location:
+|Hourly
+|Minutes
+|The number of minutes after each hour when the test runs, beginning 60 minutes after the schedule has been created
+
+|Daily
+|Time
+|The time (hour and minutes) in a day when the test runs
+
+|Weekly
+|Day of week, Time
```

Comment:
```suggestion
|Day of the week and time of day 
```
...or does the table column Required Information take values that appear onscreen or in syntax? In other words, is this capitalized and separated by a comma to indicate two separate values indicates somewhere rather than "general" narrative required info?